.github/workflows/cache-dependencies.yml

```diff
@@ -76,16 +76,49 @@ jobs:
         with: 
           repository: openmc-dev/openmc
           path: openmc
-
-      - name: Build OpenMC
+          
+      - name: Build OpenMC from scratch
         if: steps.dependencies-cache.outputs.cache-hit != 'true'
         run: |
           cd openmc
-          pip install --upgrade-strategy only-if-needed --no-binary=h5py h5py
           python tools/ci/gha-install.py
+          mkdir openmc_src
+          mkdir openmc_src/bin
+          mkdir openmc_src/lib
+          mkdir openmc_src/share
+          mkdir openmc_src/include
+          cp /usr/local/bin/openmc openmc_src/bin/.
+          cp -r /usr/local/lib/x86_64-linux-gnu/ openmc_src/lib/.
+          cp -r /usr/local/share/openmc openmc_src/share/.
+          cp -r /usr/local/share/man openmc_src/share/.
+          cp -r /usr/local/include/openmc openmc_src/include/.
+          cp /usr/local/include/pugiconfig.hpp openmc_src/include/.
+          cp /usr/local/include/pugixml.hpp openmc_src/include/.
+          cp -r /usr/local/include/fmt openmc_src/include/.
+          cp -r /usr/local/include/xtl openmc_src/include/.
+          cp -r /usr/local/include/xtensor openmc_src/include/.
+          cp -r /usr/local/include/gsl openmc_src/include/.
+          cp -r /usr/local/include/gsl-lite openmc_src/include/.
+          mv openmc_src $HOME/openmc_src
```

Comment:
Is it possible to add inline comments so somebody looking at these knows the purpose of these steps? I think it would be helpful for future group members!